### DISCUSSION TITLE
Create a chronogram with ili9488

### DISCUSSION THREAD

**Discussion by Mohamedstm** [Bodmer/TFT_eSPI#2616] [Q&A - General]
So I am trying to create a chronogram for a vector of 8 bytes, with an lcd tft ili9488 using spi with an stm32h743vit6 board, I have see in this github that there is an library for stm32 but I have encountered a lot if errors whene I tried to use it is there anyone to help understand this library and use it for my project please I really need your help.
Here is my email: mohamedfmmf20@gmail.com
Please help me

**Comment by Bodmer** [comment]
There could be many reasons for the errors. Provide more detail of what you have done, the IDE configuration, what the errors are etc.

Maybe the wrong board package is loaded? This one is reuired:
https://github.com/stm32duino/Arduino_Core_STM32

**Reply by Mohamedstm** [reply]
Do I have to download this arduino core package??
Because I just download the code from the github and then copy the file for stm32 and ili9488 and I tried using some of the functions but I did have many error like some file and functions where not defined, and I am not sure that my idea of how it actually works is right or my ideas about how to create the chronogram.
So if you want when you have time of course do you think we can make like a Google meet or something to discuss about this.
I am just a small student who want to learn and improve and make this project so please do you think it's possible??

**Reply by Bodmer** [reply]
Yes, you need that board package. Instructions to load in the Arduino IDE are on the link page.

**Reply by Mohamedstm** [reply]
Okay I will download it and try on my own but do you think that we can make that Google meet session if you have time i am not really confident that I will be able to make it.
So I really do hope that you can take a little bit of your time and accept to help me
